pkg/controller/account/byoc.go

```diff
@@ -141,10 +142,9 @@ func createBYOCAdminAccessRole(reqLogger logr.Logger, awsSetupClient awsclient.C
 
 	// Create the base role
 	roleID, croleErr := CreateRole(reqLogger, byocInstanceIDRole, principalARN, byocAWSClient, tags)
-	if err != nil {
+	if croleErr != nil {
 		return roleID, croleErr
 	}
-	reqLogger.Info(fmt.Sprintf("New RoleID: %s", roleID))
```

Comment:
Are we losing this log msg on purpose? I don't see anywhere else where we output the RoleID of the created role

Reply:
It did not seem helpful to me in the context of how our logging works within the pod.  It seemed to me to be more related to debugging than logging helpful information.

That said, I will put that line back in.

Reply:
Reverted the removal of this log message.